testFiles/Link-U/README.md

```diff
@@ -0,0 +1,232 @@
+
+[Makefile](Makefile) describes how they were created. To generate files yourself, you have to install [cavif](https://github.com/link-u/cavif) and [davif](https://github.com/link-u/davif)
+
+Additional test images may to be added. Please check [our repositories](https://github.com/link-u/avif-sample-images) for them.
+
+## hato
+
+![hato.jpg](hato.jpg)
+
+ - size: 3082x2048
+ - By Kaede Fujisaki ([@ledyba](https://github.com/ledyba))
+ - Retrieved from [her website](https://hexe.net/2017/11/27/12:27:02/).
+
+### AVIF version
+
+#### YUV 420
+
+| profile | bit depth | Monochrome | CDEF | Loop Restoration | file                                                                      |
+|---------|-----------|------------|------|------------------|---------------------------------------------------------------------------|
+| 0       | 8         |            | NO   | YES              | [here](hato.profile0.8bpc.yuv420.no-cdef.avif)                            |
+| 0       | 8         | YES        | NO   | YES              | [here](hato.profile0.8bpc.yuv420.monochrome.no-cdef.avif)                 |
+| 0       | 10        |            | NO   | NO               | [here](hato.profile0.10bpc.yuv420.no-cdef.no-restoration.avif)            |
+| 0       | 10        | YES        | NO   | NO               | [here](hato.profile0.10bpc.yuv420.monochrome.no-cdef.no-restoration.avif) |
+
+#### YUV422
+
+| profile | bit depth | Monochrome | CDEF | Loop Restroation | file                                                                      |
+|---------|-----------|------------|------|------------------|---------------------------------------------------------------------------|
+| 0       | 8         |            | NO   | YES              | [here](hato.profile2.8bpc.yuv422.no-cdef.avif)                            |
+| 0       | 8         | YES        | NO   | YES              | [here](hato.profile2.8bpc.yuv422.monochrome.no-cdef.avif)                 |
+| 0       | 10        |            | NO   | NO               | [here](hato.profile2.10bpc.yuv422.no-cdef.no-restoration.avif)            |
+| 0       | 10        | YES        | NO   | NO               | [here](hato.profile2.10bpc.yuv422.monochrome.no-cdef.no-restoration.avif) |
+| 0       | 12        |            | NO   | NO               | [here](hato.profile2.12bpc.yuv422.no-cdef.no-restoration.avif)            |
+| 0       | 12        | YES        | NO   | NO               | [here](hato.profile2.12bpc.yuv422.monochrome.no-cdef.no-restoration.avif) |
+
+## Kimono - Transformation tests
+
+[<img src="https://raw.githubusercontent.com/link-u/avif-sample-images/master/kimono.jpg" alt="kimono.jpg" height="512">](kimono.jpg)
+
+ - size: 722x1024
+ - By Momiji Jinzamomi([@momiji-san](https://github.com/momiji-san)) and Kaede Fujisaki ([@ledyba](https://github.com/ledyba))
+ - Retrieved from [their website](https://hexe.net/2018/12/24/18:59:01/).
+
+Test images for rotation(`irot`), mirroring(`imir`), cropping(`clap`).
```

Comment:
According to MIAF, "All transformative properties associated with coded and derived images required or conditionally required by this document shall be marked as essential" but that's not the case in your files. Note that I used https://gpac.github.io/ComplianceWarden-wasm/ to validate some of the files and to find this issue.

Reply:
Thanks! I re-encode all `kimono` images and mark `irot`, `imir` and `clap` as essential.
(And also, I validated those images using https://gpac.github.io/ComplianceWarden-wasm/ )